yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/AnchorConverter.java

```diff
@@ -0,0 +1,109 @@
+package com.yoti.api.client.spi.remote;
+
+import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_CHARSET;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+import com.yoti.api.client.Anchor;
+import com.yoti.api.client.SignedTimestamp;
+import com.yoti.api.client.spi.remote.proto.AttrProto;
+import com.yoti.api.client.spi.remote.proto.SignedTimestampProto;
+import com.yoti.api.client.spi.remote.util.AnchorType;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.bouncycastle.asn1.ASN1OctetString;
+import org.bouncycastle.asn1.ASN1Primitive;
+import org.bouncycastle.asn1.ASN1TaggedObject;
+import org.bouncycastle.asn1.DEROctetString;
+import org.bouncycastle.asn1.DLSequence;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class AnchorConverter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AnchorConverter.class);
+
+    Anchor convert(AttrProto.Anchor anchorProto) throws CertificateException, IOException {
+        List<X509Certificate> certificates = convertCertificates(anchorProto);
+        CertificateType certificateType = determineAnchorType(certificates);
```

Comment:
The name `CertificateType` is a little bit misleading, also we use a method that is called `determineAnchorType` to get ti back from a List of X509Certs. What do you mean for CertificateType here? the particular subset of Anchor type and value that are needed for creating the Anchor?